pytarallo/Tarallo.py

```diff
@@ -1,12 +1,16 @@
 import json
 import urllib.parse
-import requests
+import pytarallo.Errors as Errors
 from typing import Optional
+from pytarallo.Product import Product
+
+import requests
 
+from pytarallo import ProductToUpload, ItemToUpload
 from pytarallo.AuditEntry import AuditEntry, AuditChanges
-from pytarallo.Errors import *
 from pytarallo.Item import Item
```

Comment:
Same as in `Tarallo.py`, are we not already in the `pytarallo` module? 
Maybe we need to add an empty `__init__.py` directory to tell Python this is a module if it doesn't exist already